libsodium-bindings/src/LibSodium/Bindings/SealedBoxes.hs

```diff
@@ -0,0 +1,91 @@
+{-# LANGUAGE CApiFFI #-}
+{-# LANGUAGE Trustworthy #-}
```

Comment:
@Kleidukos - what's our policy on friends not letting friends use Safe Haskell?

Reply:
Deleted